pystoi/stoi.py

```diff
@@ -46,14 +46,18 @@ def stoi(x, y, fs_sig, extended=False):
             IEEE Transactions on Audio, Speech and Language Processing, 2016.
     """
     if x.shape != y.shape:
-        raise Exception('x and y should have the same length,' +
+        raise Exception('x and y should have the same length,'
                         'found {} and {}'.format(x.shape, y.shape))
 
     # Resample is fs_sig is different than fs
     if fs_sig != FS:
         x = utils.resample_oct(x, FS, fs_sig)
         y = utils.resample_oct(y, FS, fs_sig)
 
+    if min(x.shape[0], y.shape[0]) < N_FRAME:
+        raise Exception('x and y should at least {} miliseconds long'
+                        .format(int(1000 * float(N_FRAME) / float(FS))))
```

Comment:
Did you take into account the overlap-add in this calculation?

Reply:
This is the bound on the input size for the silence removal.
Thinking about it, I think it is actually included in the other check after the removal.